conda/plugins/virtual_packages/cuda.py

```diff
@@ -23,25 +26,90 @@ def cuda_version():
 
     Returns: version string (e.g., '9.2') or None if CUDA is not found.
     """
-    # platform-specific libcuda location
+    # Do not inherit file descriptors and handles from the parent process.
+    # The `fork` start method should be considered unsafe as it can lead to
+    # crashes of the subprocess. The `spawn` start method is preferred.
+    ctx = mp.get_context("spawn")
+    queue = ctx.SimpleQueue()
+    try:
+        # Spawn a subprocess to detect the CUDA version
+        detector = ctx.Process(
+            target=_cuda_driver_version_detector_target,
+            args=(queue,),
+            name="CUDA driver version detector",
+            daemon=True,
+        )
+        detector.start()
+        detector.join(timeout=60.0)
+    finally:
+        # Always cleanup the subprocess
+        detector.kill()  # requires Python 3.7+
```

Comment:
```suggestion
    # Do not inherit file descriptors and handles from the parent process.
    # The `fork` start method should be considered unsafe as it can lead to
    # crashes of the subprocess. The `spawn` start method is preferred.
    context = multiprocessing.get_context("spawn")
    queue = context.SimpleQueue()
    try:
        # Spawn a subprocess to detect the CUDA version
        detector = context.Process(
            target=_cuda_driver_version_detector_target,
            args=(queue,),
            name="CUDA driver version detector",
            daemon=True,
        )
        detector.start()
        detector.join(timeout=60.0)
    finally:
        # Always cleanup the subprocess
        detector.kill()  # requires Python 3.7+
```